tests/unit-core/test-snapshot.c

```diff
@@ -234,8 +234,27 @@ main (void)
       0x46, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
       0x14, 0x00, 0x73, 0x74, 0x72, 0x69, 0x6E, 0x67,
       0x20, 0x66, 0x72, 0x6F, 0x6D, 0x20, 0x73, 0x6E,
-      0x61, 0x70, 0x73, 0x68, 0x6F, 0x74
+      0x61, 0x70, 0x73, 0x68, 0x6F, 0x74,
     };
+
+    if (sizeof (expected_data) != snapshot_size || memcmp (expected_data, snapshot_buffer, sizeof (expected_data)))
+    {
+      printf ("Snapshot data is changed, please update tests/unit-core/test-snapshot.c.\n");
```

Comment:
`is` -> `has`

Reply:
has been

Reply:
`has been` puts more stress on the fact that _someone_ is responsible for the data change, while `has` just indicates that the data differentiates from what the test would expect.

Reply:
I vote for "has been", updated.